release/models/system/openconfig-system.yang

```diff
@@ -284,6 +290,15 @@ module openconfig-system {
         information such as the current system date and time, uptime,
         last login timestamp, etc.";
     }
+
+    leaf last-configuration-timestamp {
```

Comment:
I think this should be under [grouping system-global-state](https://github.com/openconfig/public/pull/719/files#diff-39c8ff1320ad76b4552208d599622a306cc84862f6dd94d378573c980e43e85cR353)

Reply:
Right you are, mistakenly put this under config